.github/workflows/rust.yml

```diff
@@ -36,6 +36,7 @@ jobs:
       - name: Cargo Check
         run: cargo check
       - name: Make Test-All
+        uses: taiki-e/install-action@nextest
```

Comment:
Combination of uses and run does not seem to be valid

Reply:
I see, there's no error msg being reported nor docs to understand what's the issue is. So do I need to move all makefile test commands under the `run:` section instead?

Reply:
according to https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun i might just use run to install nextest and run make test

Reply:
If u look at the CI jobs, `rust.yml` did not run

Reply:
@creativcoder Just put
```
- name: install nextest
   uses: taiki-e/install-action@nextest
```
above the cache step, leave this step as it was; it should work. To my understanding whether you can use `run` or not, depends on the plugin you are using. Here apparently it doesn't work.

Reply:
got it thanks! it's running now